sdk/customapp/csv_transformer.py

```
@@ -0,0 +1,268 @@
+            for row in csv_reader:
+                if not row:
+                    # skip empty rows
+                    continue
+                if len(row) != len(field_names):
+                    logger.info("Row %s has %d columns, expected %d",
+                                row, len(row), len(field_names))
+                # skip the first row
+                if row[0] == 'Sr No':
+                    continue
+                # convert the row to a dictionary with the field names
+                row_dict = dict(zip(field_names, row))
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)